src/config/ConfigNames.ts

```diff
@@ -179,12 +179,21 @@ class ConfigNames {
     public static readonly THUNDRA_TRACE_INTEGRATIONS_AWS_SES_MAIL_DESTINATION_MASK: string =
         'thundra.agent.trace.integrations.aws.ses.mail.destination.mask';
 
+    public static readonly THUNDRA_TRACE_INTEGRATIONS_GOOGLE_PUBLISH_BODY_MASK: string =
+        'thundra.agent.trace.integrations.google.publish.body.mask';
+
+    public static readonly THUNDRA_TRACE_INTEGRATIONS_GOOGLE_SUBSCRIPTION_BODY_MASK: string =
```

Comment:
Maybe can have a single config `THUNDRA_TRACE_INTEGRATIONS_GOOGLE_PUBSUB_MESSAGE_MASK` for both of publish and subscribe